From 3e63b264688aeda3e1da6decc8abb0d6ba6bbfc7 Mon Sep 17 00:00:00 2001
From: bwu <ben.wu@realtek.com>
Date: Wed, 22 Feb 2017 17:29:57 +0800
Subject: [PATCH 01/11] [DEV_FIX] apply patch from
 buildroot-2014.05(0001-use-gettext-0.18.patch). It is used to fix the
 following error, when using buildroot-2014.05: *** error: gettext
 infrastructure mismatch: using a Makefile.in.in from gettext version 0.17 but
 the autoconf macros are from gettext version 0.18

Change-Id: I875d387e46b205bfd3378ac99c22e4d58cda1c0a
---
 po/Makefile.in.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/po/Makefile.in.in b/po/Makefile.in.in
index fecf500..ecb2db8 100644
--- a/po/Makefile.in.in
+++ b/po/Makefile.in.in
@@ -9,7 +9,7 @@
 # General Public License and is *not* in the public domain.
 #
 # Origin: gettext-0.17
-GETTEXT_MACRO_VERSION = 0.17
+GETTEXT_MACRO_VERSION = 0.18
 
 PACKAGE = @PACKAGE@
 VERSION = @VERSION@
-- 
2.7.4

